js/markdown/shared/MarkdownCode.svelte

```diff
@@ -73,10 +73,26 @@
 			}
 		}
 	}
-	afterUpdate(() => render_html(message));
+	onMount(() => {
+		el = document.getElementById("container") as HTMLSpanElement;
+	});
```

Comment:
This is essentially the same as the binding, does just having the condition in the `afterUpdate` address this issue?

Reply:
Nope, it still crashes half the time. Seems like KaTex doesn't play well with svelte since both manipulate the DOM.